src/modules/Experience/scenes/AboutScene/components/MovingFace/MovingFace.tsx

```diff
@@ -45,13 +49,62 @@ const MovingFace = ({ scenePositionY, selectedColor }: MovingFaceProps) => {
       faceRef.current.rotation.y = mouseX * rotationIntensityY;
       //faceRef.current.lookAt(mouseX * lookAtIntensity, mouseY * lookAtIntensity, 20);
     }
+
   });
 
+  const map1 = new THREE.TextureLoader().load('https://cdn.discordapp.com/attachments/941095160517894185/1192890009741709403/note.png?ex=65aab865&is=65984365&hm=7c54f001dd572a6fc963abc396026353c22b73504b0ebfc96f6a6eac8df1d641&')
```

Comment:
What happens when this website is down??? Is there a way to retrieve this locally? Or add a safety net when is down.

To keep code organize, it is possible to move the texture you are generating to the textureLibrary ?? Or is not relevant??